mon-pix/app/routes/application.js

```diff
@@ -45,6 +45,12 @@ export default Route.extend(ApplicationRouteMixin, {
   // https://github.com/simplabs/ember-simple-auth/blob/a3d51d65b7d8e3a2e069c0af24aca2e12c7c3a95/addon/mixins/application-route-mixin.js#L132
   sessionInvalidated() {},
 
+  _setLanguage() {
```

Comment:
C'est intéressant de choisir ce nommage.
Dans l'application, on a des notions comme `FRENCH_FRANCE` et `FRENCH_SPOKEN` donc language ≠ locale.
D'ailleurs, la responsabilité de la méthode est de configurer la locale pour la session courante

Pour se rappeler que l'application ne touche qu'une session utilisateur, je propose
```suggestion
  _setSessionLocale() {
```

ou plus simplement 
```suggestion
  _setLocale() {
```